tests/dummy/app/mirage/config.js

```diff
@@ -0,0 +1,125 @@
+import Mirage from 'ember-cli-mirage';
+
+export default function() {
+  this.namespace = 'api';
+  // These comments are here to help you get started. Feel free to delete them.
+
+  /*
+    Config (with defaults).
+
+    Note: these only affect routes defined *after* them!
+  */
+  // this.urlPrefix = '';    // make this `http://localhost:8080`, for example, if your API is on a different server
+  // this.namespace = '';    // make this `api`, for example, if your API is namespaced
+  // this.timing = 400;      // delay for each request, automatically set to 0 during testing
+
+  /***********************************
+  * New Session (login) mock request
+  ***********************************/
+  this.post('session', function(db, request) {
+    var data = JSON.parse(request.requestBody);
+    var user = db.users.where({ username: data.username })[0] ||
+      db.users.where({ email: data.username })[0];
+
+    if (user && user.password === data.password) {
+      return {
+        session: {
+          key: user.sessionKey,
+          expiration: user.sessionExpiration,
+          user: {
+            id: user.id,
+            username: user.username,
+            email: user.email,
+            created_at: user.createdAt,
```

Comment:
these are API mocks as must fit the format expected by the API, so disregard this hound comment.